src/Timestamps/LargeTimestampsPlainTextFormatter.php

```diff
@@ -24,10 +24,10 @@ class LargeTimestampsPlainTextFormatter
 
     private $fmtTime;
 
-    public function __construct()
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.